core/test/unit/utils_read_csv_spec.js

```diff
@@ -0,0 +1,55 @@
+/*globals describe, it*/
```

Comment:
Up until now we have been moving files into a folder when there are enough files to justify another folder. I fully agree that it would make more sense to have the test folders reflect the normal structure so it would probably be a good idea to submit a PR that does this across the board?